src/SFA.DAS.EmployerIncentives/Application/Commands/UpdateVendorRegistrationFormCaseStatus/RefreshVendorRegistrationFormCaseStatusCommand.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace SFA.DAS.EmployerIncentives.Application.Commands.UpdateVendorRegistrationFormCaseStatus
 {
-    public class RefreshVendorRegistrationFormCaseStatusCommand : IRequest
+    public class RefreshVendorRegistrationFormCaseStatusCommand : IRequest<DateTime>
```

Comment:
Possibly over-engineering, but would it be better to return a Response type rather than a primitive type? I've seen that pattern elsewhere...

Reply:
Hi David, there is a tech debt Jira ticket to rework this whole thing :)